synthtool/gcp/templates/node_library/.nycrc

```diff
@@ -1,28 +1,22 @@
 {
   "report-dir": "./.coverage",
-  "reporter": "lcov",
+  "reporter": ["text", "lcov"],
   "exclude": [
-    "src/*{/*,/**/*}.js",
-    "src/*/v*/*.js",
-    "test/**/*.js",
-    "build/test"
```

Comment:
I notice this doesn't do anything funky with TypeScript.  Are we getting coverage from what's in `build/src`, or from `src/`?  How do we know this isn't being counted twice?

Reply:
Coverage uses source-maps, so in reports it will point the user to the files in src/ as the files missing coverage; in practice it’s always the files in build/src that are executed, but will not show up in reports due to them including source-maps.